ext/log_download.py

```diff
@@ -0,0 +1,91 @@
+from datetime import date
+from datetime import timedelta
+import urllib2 as ulib
+import logging
+from BeautifulSoup import BeautifulSoup
+import re
+import time
+import os
+
+root = logging.getLogger()
+root.setLevel(logging.DEBUG)
+
+months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sept", "Oct", "Nov", "Dec"]
```

Comment:
Would we be able to retrieve this array directly from a function of datetime library?

Reply:
calendar library provides the month names but not in shortform.